src/DotLiquid.Tests/ExtendedFilterTests.cs

```diff
@@ -74,42 +74,30 @@ public void TestTimeZone()
         {
             Context context = new Context(CultureInfo.CurrentCulture);
 
-            // Test UTC specified DateTime
```

Comment:
These tests between the type should result in a different value. On a machine NOT in UTC, converting a datetime that is EXPLICITLY local to UTC would change the hour, but a datetime that is EXPLICITLY UTC would not. This functionality is missing I guess.

Reply:
@microalps, I'm not clear what functionality is missing. Could you advise/write a test case for the scenario you're thinking of?

Reply:
Say you have a hash that includes a date - from DateTime.Now - and the local machine is not in UTC. Calling TimeZone with this parameter and 'UTC' as the destination should change the time. HOWEVER, the same with DateTime.UtcNow should not as it is already in UTC.